### DISCUSSION TITLE
"intermediate" HUD option, to see certain objects, but not others

### DISCUSSION THREAD

**Discussion by Doksha** [ppy/osu#17780] [Ideas]
In lazer, when you disable hud, *nothing* shows up, including keys and UR, and the only way to see them is to enable hud again, which is (practically) impossible while you are playing. There should at least be an option to make the hud more similar to stable, showing UR (and keys if enabled)
Maybe something more like McOsu where you can select the exact elements you want to show, with the default being just UR and keys.
I cant really play without ur, and I cant play with 20 things on my screen, so this makes it exceptionally difficult to play.
"Why not do this with skin???" because I dont want to completely disable score/acc/progress/combo, I want to be able to see that in break. There is already an option to show HUD just in break, so maybe this could stem off it.

**Comment by smoogipoo** [comment]
This is planned for skinning, you'll be able to decide which elements show up each mode. There's allusions to this around the place such as https://github.com/ppy/osu/discussions/17747.
